service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceFlightMapKeys.java

```diff
@@ -12,7 +12,7 @@ public final class WorkspaceFlightMapKeys {
   public static final String DESCRIPTION = "descriptionId";
   public static final String USER_TO_REMOVE = "userToRemove";
   public static final String ROLE_TO_REMOVE = "roleToRemove";
-  public static final String APPLICATION_ID = "applicationId";
+  public static final String APPLICATION_IDS = "applicationIds";
```

Comment:
@ddietterich is it safe to rename key for flight? I suppose they only live as long as the flight so it's probably OK? I'd suggest we keep APPLICATION_ID and add APPLICATION_IDS.

Reply:
Melissa comments this is not been used. And I also search in the whole repo, yes, no other place use it.